docs/flink_job_cluster_guide.md

```diff
@@ -0,0 +1,95 @@
+# Running Flink-Job-Cluster in Kubernetes
+
+## Overview
+
+to run a Flink-Job-Cluster, you first have to deploy the Flink-Operator. For that, follow the [user_guide](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/blob/master/docs/user_guide.md)
```

Comment:
Mention that this Helm Chart is an addition to the existing way of creating Flink job clusters. Don't need to mention deploying the operator since you will mention it in the Prerequisites section.